plugins/plugin-orion/che-plugin-orion-editor/src/main/resources/org/eclipse/che/ide/editor/orion/public/built-codeEdit-10.0/code_edit/built-codeEdit-amd.js

```diff
@@ -35907,14 +35907,35 @@ define('orion/editorCommands',[
 					var line;
 					var editor = this.editor || that.editor;
 					var model = editor.getModel();
+
 					if (data.parameters && data.parameters.valueFor('line')) { //$NON-NLS-0$
 						line = data.parameters.valueFor('line'); //$NON-NLS-0$
-					} else {
-						line = model.getLineAtOffset(editor.getCaretOffset());
-						line = prompt(messages.gotoLinePrompt, line + 1);
 						if (line) {
-							line = parseInt(line, 10);
+							editor.onGotoLine(line - 1, 0);
 						}
+						return;
+					}
+
+					line = model.getLineAtOffset(editor.getCaretOffset());
+
+					// try to use promptIDE(title, text, defaultValue, callback) function
```

Comment:
all these changes in **built-codeEdit-amd.js** will be lost after updating Orion editor to the next version